{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
```

Comment:
Sorry, I didn't mean wiki. I meant this:

http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html

This document would need updating to describe this template, and to make it applicable to all audiences and repositories.